Unittest.py

```diff
@@ -274,6 +274,44 @@ def test_pool_accuracy(self):
             for item in distribution_file['starting_items']:
                 self.assertNotIn(item, actual_pool)
 
+    def test_rutoletter_startwith(self):
+        filenames = [
+            "rutoletter-startwith",
+            "rutoletter-startwithother",
+            "rutoletter-startwithall",
+            "rutoletter-startwithall2"
```

Comment:
I think it'd be worthwhile to add a non-plentiful version of one or two of these.